src/middlewared/middlewared/api/v25_10_0/auth.py

```diff
@@ -51,38 +69,36 @@ class AuthApiKeyPlain(BaseModel):
     mechanism: Literal[AuthMech.API_KEY_PLAIN]
     username: str
     api_key: Secret[str]
-    login_options: AuthCommonOptions = Field(default=AuthCommonOptions())
+    login_options: AuthCommonOptions = Field(default_factory=AuthCommonOptions)
```

Comment:
It can be just
```suggestion
    login_options: AuthCommonOptions = AuthCommonOptions()
```
https://docs.pydantic.dev/latest/concepts/fields/#mutable-default-values